code/drasil-example/swhs/lib/Drasil/SWHS/Unitals.hs

```diff
@@ -485,37 +484,35 @@ arMax = mkQuantDef aspectRatioMax $ exactDbl 100
 
 -- Used in Constraint 5
 pcmDensityMin = mkQuantDef (ucStaged' "pcmDensityMin"
-  (nounPhraseSP "minimum density of PCM")
-  (S "the minimum density of the" +:+ phrase phsChgMtrl)
```

Comment:
here and elsewhere in this file: why undo all the uses of phrase combinators? They reveal interesting inner structure to our sentences, we don't want to lose that!